docs/exogenous/extraction-and-mining.md

```diff
@@ -0,0 +1,3 @@
+## Extraction and Mining of Fossil Fuels
```

Comment:
Sphinx seems to be handling this, but the first header should start with # (i.e. header 1), followed by ## (headear 2), followed by ### (header 3)